app/views/categories/_category.html.erb

```diff
@@ -0,0 +1,14 @@
+<%= link_to category_transactions_path(category), class:"text-dark", style:"text-decoration: none;" do %>
+  <div class="card flex-row align-items-center p-3 my-3 rounded-0">
+    <div style="width: 5rem; height: 5rem">
+      <img src=<%= category.icon %> class="card-img-top" alt=<%= category.name %>>
+    </div>
+    <div class="card-body">
+      <div class="d-flex justify-content-between">
+        <h5 class="card-title"><%= category.name %></h5>
+        <p class="card-text"><strong>$<%= category.category_total_amount %></strong></p>
+      </div>
+      <p class="position-relative" style="top: 1em"><%= category.updated_at %></p>
+    </div>
+  </div>
+<% end %>
```

Comment:
- [ ] nice job, just a few thing to adjust in your design, category icon is currently overflowing out of the category card, kindly fix that, also make your date to be in human readable format
![Screenshot from 2023-02-16 15-00-41](https://user-images.githubusercontent.com/66097371/219385926-8481ab81-ff7b-43ab-9bfa-64a55b90a3a7.png)